src/test/scala/mesosphere/marathon/tasks/InstanceTrackerImplTest.scala

```diff
@@ -53,8 +53,6 @@ class InstanceTrackerImplTest extends AkkaUnitTest {
       val originalInstance: Instance = Instance.scheduled(AppDefinition(TEST_APP_NAME))
       instanceTracker.process(Schedule(originalInstance)).futureValue
 
-      instanceTracker.process(Provision(originalInstance)).futureValue
```

Comment:
Why is that gone?

Reply:
It is not adding any value... this test is just testing that if you push an update to instance tracker and then query the instance tracker, you get that instance. I believe the provision was here (as in other places in the tests) because up until Phase 2, provision was the operation that created the instance. So to test this, now schedule is enough... This is not the level on which we would want to test all the operations...

Reply:
Ah, now I remember. Back then I was not sure which futures are called. Thanks!